torch/_inductor/graph.py

```diff
@@ -188,6 +190,12 @@ def register_buffer(self, buffer: ir.ComputedBuffer):
         self.name_to_buffer[name] = buffer
         return name
 
+    def register_work(self, work):
```

Comment:
i didn't end up using this after all; but i might still want a way to give special naming (if not handling) to work objects.  it's goofy to see code like 

```buf2 = dist.allreduce()
buf2.wait()
```